### PR TITLE
Feat/swarinfo 179/serial encoding

### PR DESCRIPTION
**Pull request by cquesnel**

### CODE REVIEW

File: HiveAR/app/src/main/java/com/swarmus/hivear/utils/CRC.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.swarmus.hivear.utils;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.IntBuffer;
+
+import static com.swarmus.hivear.utils.Serialization.int32ToBytes;
+
+public final class CRC {
```
Comment (t-constantin) [inline HiveAR/app/src/main/java/com/swarmus/hivear/utils/CRC.java:9]: static class with no constructor maybe

File: HiveAR/app/src/main/java/com/swarmus/hivear/utils/Serialization.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package com.swarmus.hivear.utils;
+
+public final class Serialization {
```
Comment (t-constantin) [inline HiveAR/app/src/main/java/com/swarmus/hivear/utils/Serialization.java:3]: static class

File: HiveAR/app/src/main/java/com/swarmus/hivear/utils/Serialization.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    private Serialization() {}
```
Comment (t-constantin) [inline HiveAR/app/src/main/java/com/swarmus/hivear/utils/Serialization.java:4]: no constructor needed

**Review by t-constantin — APPROVED** [review]
Perfect for me, small comments

File: HiveAR/app/src/main/java/com/swarmus/hivear/utils/Serialization.java (hunk 2 above)
Comment (cquesnel) [reply HiveAR/app/src/main/java/com/swarmus/hivear/utils/Serialization.java:3]: Static classes do not exist in Java https://stackoverflow.com/questions/7486012/static-classes-in-java

**Pull request merged**